imagekit-sdk/src/main/java/io/imagekit/sdk/models/results/Result.java

```diff
@@ -189,4 +190,12 @@ public String toString() {
                 ", fileType='" + fileType + '\'' +
                 '}';
     }
+
+	public Map<String, List<String>> getHeaders() {
+		return headers;
+	}
+
+	public void setHeaders(Map<String, List<String>> map) {
```

Comment:
This shouldn't be public since this isn't needed for end-users of this SDK.

Reply:
we are setting this from restClient. so setter method has to be public.